### DISCUSSION TITLE
using actions in cells?

### DISCUSSION THREAD

**Discussion by tamir-jether** [Comcast/react-data-grid#2583] [Q&A]
I want to add a button for each row, which will perform a custom action when clicked.
I was looking at the Cell Actions example, but its not really clear. And the code is using some relative imports.
Is there any clear documentation about how to add buttons to cells?

**Comment by Biswajit-Mangaraj** [comment]
Assign a function which will be returning the button you want to render to "formatter" inside action column.

**Reply by fkbeys** [reply]
> Assign a function which will be returning the button you want to render to "formatter" inside action column.

Can you show an example?
